crates/astria-conductor/src/celestia/reconstruct.rs

```diff
@@ -0,0 +1,201 @@
+use std::collections::HashMap;
+
+use astria_core::sequencer::v1::RollupId;
+use celestia_client::{
+    CelestiaRollupBlob,
+    CelestiaSequencerBlob,
+};
+use sequencer_client::tendermint::block::Height as SequencerHeight;
+use telemetry::display::base64;
+use tracing::{
+    info,
+    warn,
+};
+
+use super::{
+    verify::VerifiedBlobs,
+    ReconstructedBlock,
+};
+
+/// Reconstructs block information from verified blocks.
+///
+/// The reconstructed blocks contain a block hash, tendermint header (both extracted from
```

Comment:
it contains a `SequencerBlockHeader` now, no more tendermint :D

Reply:
Reworded. Talking about "Sequencer headers" now.